package.json

```diff
@@ -7,19 +7,12 @@
   "dependencies": {
     "@chakra-ui/icons": "^1.1.5",
     "@chakra-ui/react": "^1.8.7",
-    "@codemirror/autocomplete": "^0.19.13",
-    "@codemirror/closebrackets": "^0.19.1",
-    "@codemirror/commands": "^0.19.8",
-    "@codemirror/comment": "^0.19.1",
-    "@codemirror/fold": "^0.19.3",
-    "@codemirror/gutter": "^0.19.9",
-    "@codemirror/highlight": "^0.19.7",
-    "@codemirror/history": "^0.19.2",
-    "@codemirror/lang-python": "^0.19.4",
-    "@codemirror/language": "^0.19.7",
-    "@codemirror/panel": "^0.19.1",
-    "@codemirror/state": "^0.19.9",
-    "@codemirror/view": "^0.19.44",
+    "@codemirror/autocomplete": "^6.3.0",
+    "@codemirror/commands": "6.1.1",
+    "@codemirror/lang-python": "^6.0.2",
```

Comment:
We should review the diff for lang-python and see if any grammar changes might affect us, e.g. code structure highlighting, actual highlighting (which relies on tags in the grammar), indentation.

Reply:
Reviewed, nothing significant, though the above are all good areas to test.